sdk/build.gradle

```diff
@@ -21,6 +21,13 @@ subprojects {
             release {
                 minifyEnabled false
                 proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+                buildConfigField "String", "KLAVIYO_SERVER_URL", '"https://a.klaviyo.com"'
+            }
+            debug {
+                buildConfigField "String", "KLAVIYO_SERVER_URL", '"https://a.klaviyo.com"'
+            }
+            localDebug {
+                buildConfigField "String", "KLAVIYO_SERVER_URL", '"http://10.0.2.2:8080"'
```

Comment:
10.0.2.2 is the address on android emulators that will allow us to call in to the localhost address of our desktops. So local-klaviyo running on 127.0.0.1 can be reached within the emulator via this address